xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/objects/classes/ComputedFieldClass.java

```diff
@@ -90,6 +90,47 @@ public String getScript()
         return sValue;
     }
 
+    /**
+     * Computes and returns the raw value of this property for a given object.
+     * @param name property name
+     * @param prefix prefix to be added
+     * @param object object for which the property value has to get computed
+     * @param context current context
+     * @return the computed property value
+     */
```

Comment:
Don't forget the `@since`.

Reply:
I added it indeed in the latest commit.

Reply:
It's actually `11.8RC1` right now (but better wait for the last minute before fixing that to be sure).